src/js/components/ServiceActions.js

```diff
@@ -0,0 +1,117 @@
+import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import CosmosPackagesStore from '../stores/CosmosPackagesStore';
+import Framework from '../structs/Framework';
+import InternalStorageMixin from '../mixins/InternalStorageMixin';
+import Service from '../structs/Service';
+import UpdateConfigModal from './modals/UpdateConfigModal';
+
+const METHODS_TO_BIND = [
+  'handleEditConfigClick',
+  'handleConfigModalClose'
+];
+
+class ServiceActions extends mixin(InternalStorageMixin, StoreMixin) {
```

Comment:
`ServiceActions` sounds like an `Actions` file (those things that make requests)